### PR TITLE
LPS-172938 Modify getLocalizableLabel function to use userLanguageId

### PR DESCRIPTION
**Pull request by rebsilva**
Related Issue: https://issues.liferay.com/browse/LPS-172938

### CODE REVIEW

**Comment by rebsilva** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionContainer/ActionContainer.tsx [hunk 1]
```diff
@@ -184,6 +186,17 @@ export function ActionContainer({
 		values.objectActionExecutorKey,
 	]);
 
+	useEffect(() => {
+		const makeFetch = async () => {
+			const objectDefinition = await API.getObjectDefinitionByExternalReferenceCode(
+				objectDefinitionExternalReferenceCode
+			);
+			setCreationLanguageId(objectDefinition.defaultLanguageId);
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionContainer/ActionContainer.tsx:194]: @rebsilva Please, add a blank line between the definition of `objectDefinition` and its usage. Thanks :)

**Review by naraandrade — CHANGES_REQUESTED** [review]
Great work, @rebsilva :) Just a small change needed.

**Comment by rebsilva** [comment]
ci:stop

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionContainer/ActionContainer.tsx (hunk 1 above)
Comment (rebsilva) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionContainer/ActionContainer.tsx:194]: Will do, thanks!

**Comment by rebsilva** [comment]
Since the CI seems stuck on this PR, I will open another one with the changes requested. Thanks!

**Pull request closed without merging**